app/models/org.rb

```diff
@@ -64,6 +64,10 @@ class Org < ActiveRecord::Base
 
   belongs_to :region
 
+  has_one :tracker
+  accepts_nested_attributes_for :tracker 
+  validates_associated :tracker
```

Comment:
Nice! I did not know about this `validates_associated`.

can you add a `dependent: :destroy' to the `has_one` statement? This should prevent any orphaning of records